### PR TITLE
Teleop app not-final integration

### PR DESCRIPTION
**Pull request by dwlee**
complete integration teleop dummy

### CODE REVIEW

File: concert_teleop_app/src/concert_teleop_app/teleop_app_info.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        """
+        print "update the teleop image"
+        self.image_data = data
+        pass
+
+    def _request_teleop_cmd_vel(self, linear, angular):
+        """
+        Update the teleop image
+
+        @param cmd_vel: command of velocity
```
Comment (stonier) [inline concert_teleop_app/src/concert_teleop_app/teleop_app_info.py]: cmd_vel obso.

File: concert_teleop_app/package.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  
+  <run_depend>qt_gui_py_common</run_depend>
+  <run_depend>rqt_gui</run_depend>
+  <run_depend>rqt_gui_py</run_depend>
+  <run_depend>rqt_graph</run_depend>
+  <run_depend>rospy</run_depend>
+  <run_depend>rocon_gateway</run_depend>
+  <run_depend>rocon_utilities</run_depend>
+  <run_depend>python-rospkg</run_depend>
+
```
Comment (stonier) [inline concert_teleop_app/package.xml:19]: Scan through your python files and make sure these are correct. e.g. there is no more rocon_utilities.

**Comment by stonier** [comment]
Delete the resource icons too.

**Comment by stonier** [comment]
@dwlee awaiting the fixes before accepting this:
- delete the resource icons
- correct doc params in `_request_teleop_cmd_vel`
- package.xml depends (remove rocon_utilities and check others)

**Comment by dwlee** [comment]
I changed and fix the something as your comment.
Please review again @stonier :)
5865714..180ae20

**Comment by stonier** [comment]
- I readded the resource - the plugin exactly needs a relative icon.
- Also fixed the icon size to scale with the view.

**Pull request merged**